src/main/front/src/admin-club/pages/AdminClubSettingPage.tsx

```
@@ -0,0 +1,237 @@
+import { useParams } from "react-router";
+import { useQuery } from "@tanstack/react-query";
+import { useFetchBe } from "../../tools/api";
+
+export interface AdminClubSettingPageProps {
+  slug?: string;
+  description?: string;
+  bannerUrl?: string;
+}
+const isValidSlug = (slug: string) => /^[a-z0-9-]+$/.test(slug);
+
+function AdminClubSettingPage({}: AdminClubSettingPageProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)